docs/releases/2.1.txt

```diff
@@ -169,6 +169,9 @@ Models
 * A ``BinaryField`` may now be set to ``editable=True`` if you wish to include
   it in model forms.
 
+* The :class:`~django.db.models.functions.Replace` database function
+  replaces strings in an expression.
```

Comment:
`The...` → `The new...`

Makes this consistent with the wording used in previous release notes and https://github.com/django/django/pull/9543#discussion_r161189953.